Documentation/contributing.rst

```diff
@@ -1274,76 +1540,269 @@ simplify our Jenkinsfiles. The exported methods are:
-   to be released version always has a corresponding container image tag
-   assigned.
-2. Update the AUTHORS file by running ``make update-authors``
-3. Update the ``cilium_version`` and ``cilium_tag`` variables in
+       git checkout v1.0; git pull
+
+#. Create a branch for the release pull request:
+
+   ::
+   
+       git checkout -b pr/prepare-v1.0.3
+
+#. Update the ``VERSION`` file to represent ``X.Y.Z+1``
+#. Update the image tag versions in the examples:
+
+   ::
+   
+       make -C examples/kubernetes clean all
+
+#. Update the ``cilium_version`` and ``cilium_tag`` variables in
    ``examples/getting-started/Vagrantfile``
-4. Review all merged PRs and add ``release-note/*`` labels as necessary.
-   A useful query here is ``is:pr is:merged merged:>=2017-12-16``
-5. Generate the release notes by running.
-   ``git checkout master``, ``cd contrib/release/``,
-   ``GITHUB_TOKEN=xxxx ./relnotes --markdown-file=~/NEWS.rst v1.0.0-rc2..``
-6. Manually merge the generated file ``~/NEWS.rst`` into ``NEWS.rst`` in the
-   Cilium repository and add the title section with the corresponding release
-   date.
-7. Create a pull request with all changes above, get it merged into the
-   development branch of the to be released version.
-8. If the release is a new minor version which will receive backports, then
-   create a git branch with the name ``vX.Y``. Push this branch to GitHub and
-   protect the branch so it can't be pushed to directly to.
-9. Tag the release with the full version string ``vX.Y.Z`` and push the tag
-   to the git repository.
-10. Build all binaries and push them to S3 using ``contrib/release/uploadrev``.
-    See the ``README`` in the ``contrib/release`` directory for more information.
-11. Create a GitHub release and include the release notes as well as links to
-    the binaries.
-12. Update the release branch on
-    `Jenkins <https://jenkins.cilium.io/job/cilium-ginkgo/job/cilium/>`__ to be tested on
-    every change and Nightly.
 
+#. Update the ``AUTHORS file``
+
+   ::
+   
+       make update-authors
+
+#. Generate the ``NEWS.rst`` addition:
+
+   ::
+   
+       git shortlog v1.0.2.. > add-to-NEWS.rst
+
+#. Add a new section to ``NEWS.rst``:
+
+    ::
+
+        v1.0.3
+        ======
+
+        ::
+
+            <<contents of add-to-NEWS.rst>>
+            [...]
+            <<end of add-to-NEWS.rst>>
+
+#. Add all modified files using ``git add`` and create a pull request with the
+   title ``Prepare for release v1.0.3``. Add the label ``stable/backport`` to
+   the PR.
+
+   .. note::
+
+       Make sure to create the PR against the desired stable branch. In this
+       case ``v1.0``
+
+#. Follow standard procedures to get the 
+#. Checkout out the stable branch and pull your merged changes:
+
+   ::
+
+       git checkout v1.0; git pull
+
+#. Create a release tag:
+
+   ::
+
+       git tag -a v1.0.3 -m 'Release v1.0.3`
+
+#. Build the binaries and push it to the release bucket:
```

Comment:
How to build the binaries?

Make a note the binaries can (should) be build with the docker image and copy them from that image

Reply:
Let's do this in a follow up PR. I have never done it and the scripts we have do a full build on their own.